client/common/useKeyDownHandlers.js

```diff
@@ -38,6 +38,9 @@ export default function useKeyDownHandlers(keyHandlers) {
           /^\d+$/.test(e.code.at(-1)) ? e.code.at(-1) : e.key.toLowerCase()
         }`
       ]?.(e);
+    } else if (isCtrl && e.altKey && e.code === 'KeyN') {
+      // specifically for creating a new file
+      handlers.current[`ctrl-alt-n`]?.(e);
```

Comment:
I would stay away from `e.code` and prefer `e.key`.  It only makes a difference if the user has a non-standard keyboard layout.   `e.code` is basically "the key in the 'N' position" whereas `e.key` is "the key that types 'N'".
https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent/code
https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent/key

I guess I only built this hook to support `ctrl-` and `ctrl-shift-` and didn't do anything with `alt`.  But we want it to support any `ctrl-alt-LETTER` and not just `ctrl-alt-n`.  This should do it:

```
} else if (isCtrl && e.altKey) {
   handlers.current[`ctrl-alt-${e.key.toLowerCase()}`]?.(e);
```